test/game/gameServer.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Module dependencies.
+ */
+import mongoose from 'mongoose';
+import jwt from 'jsonwebtoken';
+import supertest from 'supertest';
+import chai from 'chai';
+import app from '../../server';
+
+const { expect } = chai;
+const server = supertest(app);
+const signupUrl = '/api/v1/auth/signup';
+const gameUrl = `/api/v1/games/${'nmtys'}/start`;
+let token; // store token for user authentication
+const expiredToken = jwt.sign({ _id: 'h123h1h2hhhhhhs' }, 'expired', { expiresIn: '0.001s' });
+const User = mongoose.model('User');
+const Game = mongoose.model('Game');
+// delete all records in User model
+User.collection.drop();
+// delete all records in Game model
+Game.collection.drop();
+
+const userDetails = {
+  name: 'Oluwadunsin Oyebiyi',
+  password: 'password',
+  email: 'ebenezer99.dawuda@gmail.com',
+
+};
+const gameData = {
+  gameID: 'nmtys',
+  gamePlayers: ['philnewman', 'temilaj', 'victoria'],
+  gameRound: 7,
+  gameWinner: 'victoria'
+};
+
+describe('Create game using authenticated route', () => {
+  before((done) => {
+    chai.request(app)
+      .post(signupUrl)
+      .send(userDetails)
+      .end((err, res) => {
+        if (err) return done(err);
+        token = res.body.token;
+        done();
+      });
+  });
+
+  it('should return 201 status code for successfully creating a game log', (done) => {
+    server
+      .post(gameUrl)
+      .set('Connection', 'keep alive')
+      .set('Accept', 'application/json')
+      .set('x-token', token)
+      .set('Content-Type', 'application/json')
+      .type('form')
+      .send(gameData)
+      .end((err, res) => {
+        expect(res).to.be.json;
+        expect(res.body.message).to.equal('Success... game logs saved');
+        expect(res.status).to.equal(201);
```

Comment:
Why are we prioritizing the status code in this test, and not including tests for the actual values like we talked about?